chsdi/views/historicalmaps.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.view import view_config
+
+from pyramid.renderers import render_to_response
+import pyramid.httpexceptions as exc
+
+
+@view_config(route_name='historicalmaps')
+def luftbilder(request):
+    width = request.params.get('width')
+    height = request.params.get('height')
+    rotation = request.params.get('rotation')
+    title = request.params.get('title')
+    bildnummer = request.params.get('bildnummer')
+    layer = request.params.get('layer')
+    lang = request.params.get('lang')
+    release_year = request.params.get('release_year')
+    baseUrl = request.registry.settings.get('geoadminhost')
+
+    if None in (title, bildnummer, layer):
```

Comment:
Is that really all the things you want to test? What about width, height, release_year etc...